lua/nvim-treesitter/parsers.lua

```diff
@@ -545,6 +545,16 @@ list.gdscript = {
   maintainers = { "@Shatur95" },
 }
 
+list.godotResource = {
+  install_info = {
+    url = "https://github.com/PrestonKnopp/tree-sitter-godot-resource",
+    files = { "src/parser.c", "src/scanner.c" },
+    requires_generate_from_grammar = true,
```

Comment:
It only requires generate from source because there has not been a recent commit. Can you ask the author whether he's planning to update to the new ABI? If the project is abandoned you could as well fork it with the source file regenerated.

Reply:
Actually I didn't even check if this was necessary. How do you know? What's the impact of having this enabled?
(sorry, getting into treesitter + nvim-treesitter is quite a run :smile: )

But yeah, I could submit a PR to the repo if necessary. Or fork, if it's dead.
I'd just need some little directions :blush:

Reply:
When you have this option on, users need the right `tree-sitter` installed. You can see the language version of the parser.c file here: https://github.com/PrestonKnopp/tree-sitter-godot-resource/blob/c611dd49e838efe25567dece14cf6353710f6d98/src/parser.c#L8 and also the tree-sitter version in the package.json if parser.c is up.to-date

Reply:
Thanks, that was very clear!
I just submitted a PR.

Let's see if it gets merged.